### PR TITLE
Audio playback control

### PR DESCRIPTION
**Pull request by rxdjina**
Added play, pause, rewind, and skip buttons

[Preview Link](https://www.dropbox.com/s/jk0dqnsa8yrb58f/RPReplay_Final1660075739.MP4?dl=0)

### CODE REVIEW

File: MetaU Capstone Project/API/SpotifyManager.m [hunk 1]
```diff
@@ -105,9 +113,78 @@ - (void)appRemote:(nonnull SPTAppRemote *)appRemote didFailConnectionAttemptWith
 - (void)playerStateDidChange:(nonnull id<SPTAppRemotePlayerState>)playerState {
     NSLog(@"Track name: %@", playerState.track.name);
     NSLog(@"player state changed");
+    
+//    [MusicSessionViewController updateMusicSession];
```
Comment (kcui) [inline MetaU Capstone Project/API/SpotifyManager.m]: Careful of commented out code!

File: MetaU Capstone Project/API/SpotifyManager.m [hunk 2]
```diff
@@ -105,9 +113,78 @@ - (void)appRemote:(nonnull SPTAppRemote *)appRemote didFailConnectionAttemptWith
...
+    
+    [self.appRemote.playerAPI skipToNext:(nil)];
+}
+
+- (void)rewindTrack {
+    NSLog(@"Rewinding music called");
+    
+    self.timestamp = 8; // Placeholder timestamp
+    
+    if (self.timestamp < MAX_MILISECONDS) { // if current timestamp < x seconds, restart current song
```
Comment (kcui) [inline MetaU Capstone Project/API/SpotifyManager.m]: you would need to create a MIN_MILLISECONDS variable right? for a lower threshold?

File: MetaU Capstone Project/View Controllers/MusicSessionViewController.m [hunk 3]
```diff
@@ -268,26 +236,34 @@ - (NSString *)getInfo: (NSString *)objectId {
...
 
+    __block NSDictionary *dataRecieved = [[NSDictionary alloc] init];
     [[[NSURLSession sharedSession] dataTaskWithRequest:request completionHandler:
       ^(NSData * _Nullable data,
         NSURLResponse * _Nullable response,
         NSError * _Nullable error) {
 
-          NSString *myString = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
-          NSLog(@"Data received: %@", myString);
+        NSString *strISOLatin = [[NSString alloc] initWithData:data encoding:NSISOLatin1StringEncoding];
```
Comment (kcui) [inline MetaU Capstone Project/View Controllers/MusicSessionViewController.m:256]: add a comment for why you need to encode this way!

**Review by kcui — APPROVED** [review]
Great work! Really digging the icons :)

**Review by Chrishon-Meta — APPROVED** [review]

**Pull request merged**